cg/cli/set/cases.py

```diff
@@ -24,12 +22,12 @@ def _get_samples_by_identifiers(identifiers: click.Tuple([str, str]), store: Sto
 def _get_cases(identifiers: click.Tuple([str, str]), store: Store) -> List[Family]:
     """Get cases that have samples that match identifiers if given"""
     samples_by_id: List[Sample] = _get_samples_by_identifiers(identifiers, store)
-    cases: Set[Family] = set()
+    case_list: Set[Family] = set()
     for sample in samples_by_id:
         for link in sample.links:
-            cases.add(link.family)
+            case_list.add(link.family)
 
-    return list(cases)
+    return list(case_list)
```

Comment:
we typically adopt the style of using:
`case` for a single case
`cases` for a List[Family]

so how it was before is what is done for other models